pkg/azure/api/api.go

```diff
@@ -61,6 +65,7 @@ func NewClient(subscriptionID, resourceGroup string, metrics MetricsAPI, rateLim
 		resourceGroup:   resourceGroup,
 		interfaces:      network.NewInterfacesClient(subscriptionID),
 		virtualnetworks: network.NewVirtualNetworksClient(subscriptionID),
+		vms:             compute.NewVirtualMachineScaleSetVMsClient(subscriptionID),
```

Comment:
mind naming this `vmssVMs` since we'll have to add support for plain VMs later as well?